mlir/docs/Dialects/ArmSME.md

```diff
@@ -1,13 +1,28 @@
 # 'ArmSME' Dialect
 
-[TOC]
+Basic dialect to target Arm SME.
+
+This dialect defines custom and LLVM IR intrinsic operations that are used to
+target Arm Scalable Matrix Extension. Through the available lowerings one can,
```

Comment:
nit: Maybe just say passes? I'm not sure if `-allocate-arm-sme-tiles` is a lowering, but it is used to target SME
```suggestion
target Arm Scalable Matrix Extension. Through the available conversion and ArmSME passes one can,
```